uber/models.py

```diff
@@ -615,8 +615,8 @@ def change_badge(self, attendee, badge_type, badge_num=None):
             return 'Badge updated'
 
         def everyone(self):
```

Comment:
Last I checked it was used in summary.py ... which explains why that page has so many load problems.